package/Shaders/ISReflectionsRayTracing.hlsl

```diff
@@ -48,18 +50,42 @@ float3 ViewToUVDepth(float3 view)
 	return float3(0.5 * view.x + 0.5, 0.5 - 0.5 * view.y, view.z);
 }
 
+// Simple hash function for generating a pseudo-random float
+float hash(float2 p)
+{
+	p = frac(p * float2(0.1031, 0.1030));      // Random values for perturbation
+	p *= dot(p, p.xy + float2(33.33, 33.33));  // Mix values
+	return frac(p.x * p.y * float2(0.5, 0.5).x);
+}
+
 PS_OUTPUT main(PS_INPUT input)
 {
 	PS_OUTPUT psout;
 	psout.Color = 0;
 
+#	ifndef ENABLESSR
+	// Disable SSR raymarch
+	return psout;
+#	endif
+#	ifdef VR
+	uint eyeIndex = input.TexCoord.x >= 0.5;
```

Comment:
Can't you use `GetEyeIndexFromTexCoord` here instead?

Reply:
Yes. Good point.